### ISSUE TITLE
No Flag Validation

### ISSUE THREAD

**Issue by aneeshkp** [redhat-nfvpe/helm2go-operator-sdk#12]
**Describe the bug**
`go run main.go convert --api-version=apps.example.com/v1aplha1 --helm-chart test/bitcoind/`

doesn't ask for 
  `--output-dir string           Directory to put the new go operator`
or
`--kind string                 Kubernetes CustomResourceDefintion kind. (e.g AppService)`

**To Reproduce**
`go run main.go convert --api-version=apps.example.com/v1aplha1 --helm-chart test/bitcoind/`

**Expected behavior**
Validates required flags.

**Screenshots**
```shell
2019-07-02 13:47:10] 🤠 Converting Existing Helm Chart test/bitcoind/ to Go Operator!
[2019-07-02 13:47:10] Resource Type *v1beta2.Deployment is Unsupported! Update API Version to Continue.

[2019-07-02 13:47:10] Resource Type *v1.PersistentVolumeClaim is Unsupported! Update API Version to Continue.

[2019-07-02 13:47:10] open pkg/controller/_controller.go: no such file or directory

false
[2019-07-02 13:47:10] 🤠 Go Operator Can Be Found At:            
```

**Comment by sjakati98** [comment]
Addressed in commit 272160e70250dd34de0b9b93a419812933998378

**Comment by aneeshkp** [comment]
Fixed
